LiveLines.Lines/LinesService.cs

```diff
@@ -33,6 +36,37 @@ public async Task<Line> CreateLine(LoggedInUser loggedInUser, LineToCreate lineT
             ? await _songService.AddSong(lineToCreate.SpotifySongId)
             : null;
 
-        return await _linesStore.CreateLine(loggedInUser, lineToCreate.Body, songId);
+        return await _linesStore.CreateLine(loggedInUser, lineToCreate.Body, songId, lineToCreate.ForYesterday);
+    }
+
+    public async Task<LineOperations> GetLineOperations(LoggedInUser loggedInUser)
+    {
+        bool canPostToday = false;
+        bool canPostYesterday = false;
+
+        DateTime today = DateTime.Today;
+
+        try
+        {
+            await _linesStore.GetLineForDate(loggedInUser, today);
+        }
+        catch (LinesStoreException)
+        {
+            canPostToday = true;
+
+            if (DateTime.Now.Hour < 12)
+            {
+                try
+                {
+                    await _linesStore.GetLineForDate(loggedInUser, today.AddDays(-1));
+                }
+                catch (LinesStoreException)
+                {
+                    canPostYesterday = true;
+                }
```

Comment:
This try catch block feels strange, since being able to post yesterday when we're failing a request isn't exceptional, it's going to happen a fair amount and is expected. Perhaps getting the two latest date times for lines for a user would make this operation easier?